lectures/heavy_tails.md

```diff
@@ -1020,9 +1067,7 @@ Since $r \geq \alpha$, we have $\mathbb E X^r = \infty$.
 ```{exercise}
 :label: ht_ex3
 
-Repeat exercise 1, but replace the three distributions (two normal, one
-Cauchy) with three Pareto distributions using different choices of
-$\alpha$.
+Refer to the figure in the section titled "Simulations". Your task is to replicate this figure, but instead of using normal and Cauchy distributions, use three Pareto distributions with different choices of $\alpha$.
```

Comment:
Break this into two sentences:

Refer to the figure in the section titled "Simulations". 

Your task is to replicate this figure, but instead of using normal and Cauchy distributions, use three Pareto distributions with different choices of $\alpha$.